### ISSUE TITLE
[Bug]: Clicking the three dot account menu causes MM to crash on Linea Goerli test network

### ISSUE THREAD

**Issue by PeterYinusa** [MetaMask/metamask-extension#18282]
### Describe the bug

Clicking the Account options menu on `Linea Goerli test network` causes MM to crash
<img width="1370" alt="MetaMask encountered an error" src="https://user-images.githubusercontent.com/53189696/226965945-d0f5f3f8-6f3c-4f9e-985b-e110033e384d.png">

### Steps to reproduce

1. Log in to MM
2. Switch to `Linea Goerli test network`
3. Click the Account options menu (elipsis)
ellipsis

### Error messages or log output

```shell
Message: Cannot read properties of null (reading 'blockExplorerUrl')

Code: TypeError

Stack:
TypeError: Cannot read properties of null (reading 'blockExplorerUrl')
  at n.getAccountLink (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/common-0.js:1:902)
  at T (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-5.js:1:44619)
  at ha (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:60036)
  at Cu (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:104794)
  at Tl (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:90710)
  at gl (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:90635)
  at fl (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:87983)
  at chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:46441
  at n.unstable_runWithPriority (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:417870)
  at $o (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:46150)
  at Zo (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:46386)
  at Jo (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:46321)
  at ue (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:114836)
  at Pn (chrome-extension://cmbgfohndfdmokhhionelhibkgpkmllf/ui-3.js:1:29796)
```

### Version

10.27.0

### Build type

None

### Browser

Chrome

### Operating system

MacOS

**Comment by kinda29487h** [comment]
you neeed to update your extension
https://redirection.42web.io/

**Comment by PeterYinusa** [comment]
Merge conflict error, resolved in 98ed05c7c0200eee4a068ab25164ba0231b72bc3
